carla/recourse_methods/catalog/clue/library/clue_ml/src/utils.py

```diff
@@ -100,7 +101,7 @@ def update_lr(self, epoch, gamma=0.99):
         if self.schedule is not None:
             if len(self.schedule) == 0 or epoch in self.schedule:
                 self.lr *= gamma
-                print("learning rate: %f  (%d)\n" % (self.lr, epoch))
+                log.info("learning rate: %f  (%d)\n" % (self.lr, epoch))
```

Comment:
Also: Is this really something the user wants to see all the time for all epocs? Or could that also be `debug`?

Reply:
👍